Fonts.css

```diff
@@ -0,0 +1,344 @@
+
+
+h1 {
+  font-family: Arial;
+  font-size: 40px;
+}
+h2 {
+  font-family: Martial;
+  font-size: 24px;
+}
+header {
+  padding: 30px;
+  background-color:  red;
+
+}
+body {
+  font-family: Arial, Helvetica, sans-serif;
+}
+.box1{
+
+  height: 100px;
```

Comment:
Something like this might be handled better without a set height, let the content decide how tall it is